Tools/scripts/build_options.py

```diff
@@ -106,4 +106,14 @@ def __init__(self, category, label, define, description, default, dependency):
     Feature('Other', 'AIRSPEED', 'AP_AIRSPEED_ENABLED', 'Enable Airspeed Sensors', 1, None),    # Default to enabled to not annoy Plane users
     ]
 
+    Feature('GPS Drivers', 'SBP2', 'AP_GPS_SBP2_ENABLED', 'Enable SBP2 GPS', 0, 'SBP'),
```

Comment:
Does SBP2 really depend on SBP?!  Should it?

Reply:
it uses the same detect logic..fixing that is beyond scope